workflow/commit-messages.md

```diff
@@ -0,0 +1,34 @@
+# Commit Guidelines
+
+As a general guideline you can follow the idea that *a single commit should enclose a single problem/feature that has been fixed/added*.
+
+This way of organizing your changes to the codebase makes is easy for reviewers and people who try to understand the history of a project to determine what has been happening in the project.
+
+Here a both a good and a bad example of of commit message.
+
+### BAD
```

Comment:
Here, you're using `<h3>`s under an `<h1>` above. This is usually a sign that the documents structure could be reconsidered. In this case, I think maybe "Good" and "Bad" shouldn't be headings (and probably shouldn't be in all capitals). In this case, I think just:

``` markdown
Good:

{example}

Bad:

{example}
```

would have been fine. Or maybe even just write it as a sentence:

> For example, "Fix bug where database gets deleted on POST request" is better than "fixes bug".